docker-compose.yaml

```diff
@@ -1,7 +1,7 @@
-version: '3.4'
+version: "3.4"
 
 services:
-  app:
+  php:
```

Comment:
The part about volumes for Linux should be synced from api-platform/api-platform too.

Reply:
I'm not sure, should we delete the lines?